packages/overlay/src/PlacementController.ts

```diff
@@ -267,13 +267,17 @@ export class PlacementController implements ReactiveController {
         }
     }
 
-    public resetOverlayPosition = (): void => {
-        if (!this.target || !this.options) return;
-
+    public clearOverlayPosition(): void {
         this.target.style.removeProperty('max-height');
-        this.target.style.removeProperty('height');
+        this.target.style.removeProperty('max-width');
```

Comment:
Little nit, but this pattern looks a little fragile (especially since these are set with intercap naming). I'm not sure if this is a frequent pattern in the library, but if it is, it might be worth adding a utility that makes the reset of style overrides more automatic. So e.g.


```
// To set overrides
this.targetOverrides = new StyleOverride(target, { maxWidth: ..., maxHeight: .... })`

// To clear
this.targetOverrides.reset()
```

Reply:
This is a nice looking pattern. If we were doing this elsewhere I'd definitely pick up a version of this to simplify things!